code/game/machinery/bots/cleanbot.dm

```diff
@@ -381,31 +380,29 @@ text("<A href='?src=\ref[src];operation=oddbutton'>[src.oddbutton ? "Yes" : "No"
 		. = ..()
 
 /obj/machinery/bot/cleanbot/roomba/Crossed(atom/A)
-	if(coolingdown)
-		return
 	if(isliving(A))
 		var/mob/living/L = A
-		if(prob(10))
-			annoy(L)
+		annoy(L)
 		..()
 
 /obj/machinery/bot/cleanbot/proc/attack_cooldown()
 	coolingdown = TRUE
-	spawn(attack_cooldown)
-		coolingdown = FALSE
+	spawn(1 SECONDS)
+	coolingdown = FALSE
```

Comment:
i think you missed an intendation here

Reply:
it works either way but it came to me in my sleep that I should probably fix it